crypto/rand/drbg_lib.c

```diff
@@ -1090,10 +1096,13 @@ static int drbg_add(const void *buf, int num, double randomness)
          * inevitably. So we use a trick to mix the buffer contents into
          * the DRBG state without forcing a reseeding: we generate a
          * dummy random byte, using the buffer content as additional data.
+         * Note: This won't work with RAND_DRBG_FLAG_CTR_NO_DF.
```

Comment:
Correct. Although it would be quite daring to configure no_df in the absence of an entropy source ;-)